src/components/dashboard/announcements.tsx

```diff
@@ -57,12 +96,15 @@ const Announcements = (): JSX.Element => {
         }
         title={<h3>Announcements</h3>}
       />
-      <MainAnnouncementItem
-        name="Shortened periods from period 4 onwards"
-        desc="Shortened periods due to a school-wide teacher meeting at 16:00"
-        postDate={new Date(2021, 9, 12)}
-        source="https://youtu.be/dQw4w9WgXcQ"
-      />
+      <ul>
+        <MainAnnouncementItem
```

Comment:
can you be certain that there would be at least an announcement being inputted
Should there be a base case if there're no announcement

Reply:
WHY HIGHLIGHTING NEVER WORK